nifi-commons/nifi-processor-utilities/src/main/java/org/apache/nifi/processor/util/StandardValidators.java

```diff
@@ -26,6 +26,8 @@
 import java.util.concurrent.TimeUnit;
 import java.util.regex.Pattern;
 
+import oi.thekraken.grok.api.Grok;
```

Comment:
This validation roytine should not be added to standard validators in order to avoud impirting grok into the standard validator